app/models/manageiq/providers/kubernetes/container_manager/refresh_parser.rb

```diff
@@ -362,8 +362,33 @@ def parse_container(container, pod_id)
         :backing_ref     => container.containerID,
         :container_image => parse_container_image(container.image, container.imageID)
       }
-
-      # TODO, state
+      Hash(container.state).each do |key, val|
```

Comment:
Yeah, that's what I thought. Do the thing that I said.

(Why are you calling `Hash()`?)